<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContactDetails</class>
 <widget class="QDialog" name="ContactDetails">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>419</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Contact</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="3" column="0">
    <widget class="QFrame" name="frDesktopControls">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="1">
       <widget class="QPushButton" name="btSave">
        <property name="text">
         <string>Save</string>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="btCancel">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QFrame" name="frBBControls">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QPushButton" name="btCancelBB">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QPushButton" name="btSaveBB">
        <property name="text">
         <string>Save</string>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="bbFrameLabel">
        <property name="text">
         <string>Edit Contact</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="3">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QScrollArea" name="saContactDetails">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>400</width>
        <height>347</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Contact details</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="2" colspan="2">
           <layout class="QFormLayout" name="layContactDetails">
            <property name="fieldGrowthPolicy">
             <enum>QFormLayout::ExpandingFieldsGrow</enum>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>First Name:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="ExtLineEdit" name="leFirstName"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Last Name:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="ExtLineEdit" name="leLastName"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Company:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="ExtLineEdit" name="leCompany"/>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="lbPhone">
              <property name="text">
               <string>Phone number:</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="ExtLineEdit" name="lePhone"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ExtLineEdit</class>
   <extends>QLineEdit</extends>
   <header>extlineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>btSave</sender>
   <signal>clicked()</signal>
   <receiver>ContactDetails</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>293</x>
     <y>277</y>
    </hint>
    <hint type="destinationlabel">
     <x>124</x>
     <y>296</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btCancel</sender>
   <signal>clicked()</signal>
   <receiver>ContactDetails</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>388</x>
     <y>278</y>
    </hint>
    <hint type="destinationlabel">
     <x>292</x>
     <y>296</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btCancelBB</sender>
   <signal>clicked()</signal>
   <receiver>ContactDetails</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>64</x>
     <y>27</y>
    </hint>
    <hint type="destinationlabel">
     <x>104</x>
     <y>43</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btSaveBB</sender>
   <signal>clicked()</signal>
   <receiver>ContactDetails</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>371</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>342</x>
     <y>43</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lePhone</sender>
   <signal>returnPressed()</signal>
   <receiver>ContactDetails</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>164</x>
     <y>165</y>
    </hint>
    <hint type="destinationlabel">
     <x>121</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leFirstName</sender>
   <signal>returnPressed()</signal>
   <receiver>leLastName</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>141</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>133</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leLastName</sender>
   <signal>returnPressed()</signal>
   <receiver>leCompany</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>225</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>225</x>
     <y>136</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leCompany</sender>
   <signal>returnPressed()</signal>
   <receiver>lePhone</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>308</x>
     <y>138</y>
    </hint>
    <hint type="destinationlabel">
     <x>305</x>
     <y>165</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
